worker/src/core/loaders/message-worker/index.ts

```diff
@@ -91,10 +91,20 @@ const sendMessage = tracer.wrap(
       'resource.name': 'sendMessage',
     },
   },
-  (message: Message): Promise<void> => {
+  (
+    message: Message,
+    metadata: {
+      campaignType: string
```

Comment:
based on what we discussed, I need to find a way to get Datadog to specially monitor the `campaignType` field if I want to run analytics on it using Datadog's dashboard right 🤔

@stanleynguyen